sidekick_core/lib/sidekick_core.dart

```diff
@@ -39,6 +39,7 @@ export 'package:sidekick_core/src/git.dart';
 export 'package:sidekick_core/src/repository.dart';
 export 'package:sidekick_core/src/sidekick_package.dart';
 export 'package:sidekick_core/src/template/sidekick_package.template.dart';
+export 'package:sidekick_core/src/version_checker.dart';
```

Comment:
Instead of exporting `VersionChecker` to make `getLatestDependencyVersion` available in `package:sidekick`, I'd copy those 10 LOC over.

We could also use https://pub.dev/documentation/pub_updater/latest/pub_updater/PubUpdater/getLatestVersion.html instead

Reply:
Or just expose the `getLatestDependencyVersion` function and split it from `VersionChecker`